src/main/java/eu/bbmri_eric/negotiator/dto/discoveryservice/MolgenisCollection.java

```diff
@@ -7,9 +7,14 @@
 @Getter
 @AllArgsConstructor
 @NoArgsConstructor
-public class MolgenisCollection {
+public class MolgenisCollection implements GenericResource {
```

Comment:
wouldnt it make sense to remove the lombok generated getters and setters for these classes?

Reply:
Actually they are used when retrieving data from Directory (it is converted into these internal objects). I would leave them for convenience, also considering this is a temporary solution that will disappear once that EMX2 support will be implemented